README.md

```diff
@@ -1 +1,41 @@
-# ToDo-List
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Awesome-Books-ES6
+
+> ### In this project, I built a simple todo list App.
+
+
+## Built With
+
+- HTML - HTML5
+- CSS - CSS3
+- JavaScript 
+
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+`git clone git@github.com:Bondok6/ToDo-list.git`
```

Comment:
- [ ] In the `Getting Started` section in order to make it more clear to the users please add `npm install` to install packages and `npm start` to start the server commands.